libs/mescal/src/mescal/agave_de_v2/app_listings.clj

```diff
@@ -35,6 +35,7 @@
      :pipeline_eligibility {:is_valid true :reason ""}
      :rating               {:average 0.0 :total 0}
      :step_count           1
+     :permission           "read"
```

Comment:
Agave apps can't currently be modified from within the DE, so Agave apps always have `read` permission in the app listing.